index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html class="no-js">
+<!-- Meta Tags -->
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
+    <title></title>
+    <meta name="description" content="HTML template by Rima"/>
+    <meta name="keywords" content="HTML5, CSS3, JavaScript, Jquery"/>
+    <meta name="author" content="Rima"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Favicon -->
+    <link rel="shortcut icon" href="images/favicon.png">
+
+    <!--	Stylesheets 
+    <link  rel="stylesheet" href="css/font-awesome.min.css"/>-->
+    <link rel="stylesheet" href="css/bootstrap.css" media="all">
+    <link rel="stylesheet" href="css/bootstrap.min.css" media="all">
+    <link rel="stylesheet" href="css/custom.css" media="all">
+    <link rel="stylesheet" href="css/main.css" media="all">
+
+
+    <!-- jQuery 
+        <script type='text/javascript' src='js/vendor/jquery-1.10.2.min.js'></script>
+        <script type="text/javascript" src="js/vendor/jquery-ui.js"></script>
+        <script type="text/javascript" src="js/vendor/jquery.browser.min.js"></script> -->
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-md-1 col-lg-1"></div>
+        <header class="col-md-10 col-lg-10">
+            <div class="logo">
+                <img src="images/logo.png" width="50" height="50">
+                <span class="logo_text">DISPLAY</span>
+            </div>
+            <div class="social_media">
+                <span class="twitter"><img src="images/twitter.png"></span>
```

Comment:
Whenever you see list of items, its best practice to use List both Social Media icons & Navigation menu.try to avoid span for list of items.
<ul>
<li><a href=" "></a></li>
<li><a href=" "></a></li>
<li><a href=" "></a></li>
</ul>